src/test/groovy/org/gradle/android/JdkImageWorkaroundTest.groovy

```diff
@@ -257,4 +257,71 @@ class JdkImageWorkaroundTest extends AbstractTest {
         buildResult.task(':library:compileDebugJavaWithJavac').outcome == TaskOutcome.FROM_CACHE
         buildResult.task(':library:compileReleaseJavaWithJavac').outcome == TaskOutcome.FROM_CACHE
     }
+
+    def "jdkImage is normalized when using different JDK version in the build and toolchain configuration"() {
+
+        def zuluPath = System.getProperty(ZULU_PATH)
```

Comment:
Why is is important to use the zulu JDK here and not the one defined by `JAVA_HOME`? In the action they are both defined to be using `JDK 11`.

Reply:
good point, technically we don't need it. Removed.